packages/data-service/src/instance-detail-helper.js

```diff
@@ -70,7 +72,7 @@ function parseBuildInfo(buildInfo) {
     max_bson_object_size: buildInfo.maxBsonObjectSize,
     enterprise_module: getMongoDBBuildInfo.isEnterprise(buildInfo), // Cover both cases of detecting enterprise module, see SERVER-18099.
     query_engine: buildInfo.queryEngine ? buildInfo.queryEngine : null,
-    raw: buildInfo // Save the raw output to later determine if genuine MongoDB
+    raw: buildInfo, // Save the raw output to later determine if genuine MongoDB
```

Comment:
Same here, no trailing commas. Or do you want to introduce this style?

Reply:
That is the prettier style yeah .. @alenakhineika would you rather revert these changes?

Reply:
Yeah, I think it doesn't match the team style for other projects and, makes PR unnecessarily huge, and it would be easier to review really relevant changes related to introducing typescript.